stories/theme/index.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  Title,
+  Description,
+  Heading,
+  Source,
+  Stories,
+} from '@storybook/addon-docs/blocks';
+import type { Meta } from '@storybook/react';
+import { BREAKPOINTS } from 'src/constants';
+
+export * from './boxShadows.stories';
+export * from './colors.stories';
+export * from './fonts.stories';
+export * from './typography.stories';
+
+const styledComponentSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  'export const StyledComponent = styled.div`\n  color: ${({ theme }) => theme.COLORS.primary};\n`';
+
+const cssPropSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  '<Component css={(theme: ThemeType) => css`color: ${theme.COLORS.primary}`}>';
+
+export default {
+  title: 'Constants/Theme',
+  parameters: {
+    chromatic: { viewports: [BREAKPOINTS.xs] },
+    docs: {
+      page: () => (
+        <React.Fragment>
+          <Title />
+          <Description />
+          <Heading>Usage:</Heading>
+          <Description>
+            The `theme` object is accessible (in theme aware apps using
```

Comment:
I think we want to make this description as comprehensive as possible (without running the risk of going out-of-date), so that someone with no context of emotion can look to this page and see all the ways we use it in our apps. 

For usage, we should cover the following scenarios:

1. `styled` as prop (currently covered)
1. `css` as function parameter (currently covered)
1. `withTheme/useTheme`
    - The three above usages can probably just point to the docs: https://emotion.sh/docs/theming#usage
1. `<Global styles={(theme) => {}} />`, the docs note "It accepts a styles prop which accepts the same values as the css prop except it inserts styles globally": https://emotion.sh/docs/globals

I would also considering pointing to the docs to show what a "theme-aware app" is (code wrapped in a `ThemeProvider` that has provided a `theme`: https://emotion.sh/docs/theming#themeprovider-reactcomponenttype)

Reply:
yep revamped that section a bit, could be updated further in next prs